shell/client/00-startup.js

```diff
@@ -14,6 +14,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+if ('serviceWorker' in navigator) {
+	window.addEventListener('load', function() {
```

Comment:
Tabs -> spaces

Reply:
@zenhack Your Tabs -> spaces commit didn't touch this file, so I assume this isn't resolved here.